src/app.js

```diff
@@ -41,11 +37,14 @@ export default class App extends Component {
         <GordonHeader onDrawerToggle={this.onDrawerToggle} onSignOut={this.onAuthChange} />
         <GordonNav onDrawerToggle={this.onDrawerToggle} drawerOpen={this.state.drawerOpen} />
         <main className="app-main">
-          <Route exact path="/" component={Home} />
-          <Route exact path="/about" component={About} />
-          <Route path="/activities" component={ActivitiesAll} />
-          <Route path="/activity/:activityId/edit" component={ActivityEdit} />
-          <Route path="/help" component={Help} />
+          {routes.map(route => (
+            <Route
+              key={route.path}
```

Comment:
Good question! The `key` is a required attribute and must be unique for each route. I would use the array index of the routes, but our ESLint rules say to not do that (I didn't look into it, but I'm sure they have a good reason for it). So, since paths should pretty much always be unique (technically you can have two routes with the same path, but we probably will not use that feature), they seem like the best candidate for a unique key in this situation (short of generating a unique key somewhere else).